eth/beacon/types/deposits.py

```diff
@@ -0,0 +1,98 @@
+from typing import (
+    Sequence,
+)
+
+from eth_typing import (
+    Hash32,
+)
+import rlp
+from rlp.sedes import (
+    CountableList,
+)
+
+from eth.rlp.sedes import (
+    hash32,
+    uint64,
+    uint256,
+)
+
+
+class DepositParameters(rlp.Serializable):
```

Comment:
I'd suggest we make a new file for `SlashableVoteData`, it will be easier to look up the data structure.